tests/Unit/data/migrations_using_drop_column/2023_08_07_000000_create_add_drop_to_users_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Unit\MigrationsUsingDropColumn;
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAddDropToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('users', static function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('name')->nullable();
+            $table->string('email')->unique();
+            $table->timestamps();
+        });
+
+        Schema::table('users', static function (Blueprint $table) {
+            $table->string('country')->nullable();
+            $table->string('city')->nullable();
```

Comment:
I think it would be better if this was also in the previous block. That way, we can be extra sure `dropColumn` works as expected.

Reply:
Done.

Reply:
Actually 8852c3cbbb074bf9d0f493f5b7d1da6d172b3590 was enough. We don't need the lines 27 to 30 now.

Reply:
Roger! Reverted.